src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -0,0 +1,401 @@
+﻿namespace Equinox.MessageDb
+
+open System
+open System.Text.Json
+open System.Threading.Tasks
+open Equinox.Core
+open FSharp.Control
+open FsCodec
+open Npgsql
+open Serilog
+
+type EventBody = JsonElement
+type EventData = IEventData<JsonElement>
```

Comment:
not sure this is a good alias to expose, and not sure it helper internal readability either (the first one is fine)